sdjournal/read.go

```diff
@@ -188,6 +188,29 @@ process:
 	return
 }
 
+func (r *JournalReader) FollowAsync(until <-chan time.Time, recieve chan<- []byte) <-chan error {
+	errCh := make(chan error, 1)
+	go func() {
+		var msg = make([]byte, 64*1<<(10))
+		for {
+			c, err := r.Read(msg)
+			if err != nil && err != io.EOF {
```

Comment:
Hmm, you're still going to need to implement the logic @ https://github.com/coreos/go-systemd/blob/7b2428fec40033549c68f54e26e89e7ca9a9ce31/sdjournal/read.go#L156 , otherwise it's going to spin wildly at EOF again.